zerver/lib/bugdown/__init__.py

```diff
@@ -1101,6 +1111,7 @@ def extendMarkdown(self, md, md_globals):
                         \*\*                         # ends by double asterisks
                        """
         md.inlinePatterns.add('stream', StreamPattern(stream_group), '>backtick')
+        md.inlinePatterns.add('katex', KatexPattern(r'(?P<delim>\${1,2})(?P<body>[^$]+)(?P=delim)'), '>backtick')
```

Comment:
I think this line is a little long. Maybe you could add a line break after `KatexPattern(`, similar to line 1116?